### PR TITLE
chore: dark mode changes

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

We don't support dark mode everywhere in the app yet

## Changes

|Before|After|
|----|----|
| Screenshot 2023-11-03 at 15 16 16 | Screenshot 2023-11-03 at 15 15 05 |

|Before|After|
|----|----|
| Screenshot 2023-11-03 at 15 17 50 | Screenshot 2023-11-03 at 15 53 00 |

|Before|After|
|----|----|
| Screenshot 2023-11-03 at 16 13 45 | Screenshot 2023-11-03 at 16 07 10 |

## How did you test this code?

Visual changes

### CODE REVIEW

File: frontend/__snapshots__/components-cards-text-card--template.png
Comment (thmsobrmlr) [inline frontend/__snapshots__/components-cards-text-card--template.png:1]: Somehow these cards got broken. I've ran storybook locally and the issue is also present there, so not just a test fluke,

**Review by thmsobrmlr — COMMENTED** [review]
Thanks for cleaning up this! Somethings seems to have broken the text cards, otherwise looks good.

File: frontend/__snapshots__/components-cards-text-card--template.png
Comment (daibhin) [reply frontend/__snapshots__/components-cards-text-card--template.png:1]: Thanks for pointing it out. Tracked down that there was some unrelated CSS in the file I'd deleted. Moved it into `TextCard.scss` and everything works again
Screenshot 2023-11-06 at 12 04 27

**Review by thmsobrmlr — APPROVED** [review]
🙌

**Pull request merged**